### PR TITLE
Webpack production build

### PR DESCRIPTION
**Pull request by Matt-Butler**
### Summary
Updating the script for the webpack production build. 

Here is more information about what the production flag does:
https://webpack.js.org/guides/production/#the-automatic-way

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: bundle exec puma -C config/puma.rb
```
Comment (Matt-Butler) [inline Procfile]: I took this Procfile from here:
https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/Procfile

This file needs to be added for production deployments with react_on_rails: https://github.com/shakacode/react_on_rails/blob/master/docs/additional-reading/heroku-deployment.md#heroku-deployment

**Comment by emilyrohrbough** [comment]
Are you adding a Heroku deployment for terra-ui? I noticed you added a Procfile.

**Comment by Matt-Butler** [comment]
> Are you adding a Heroku deployment for terra-ui? I noticed you added a Procfile.

Right now, we are manually pushing to heroku using the cli. Adding the procfile precompiles my assets when pushing to heroku.

**Comment by Matt-Butler** [comment]
https://github.com/cerner/terra-ui.com/issues/13

**Pull request merged**